src/runtime/components/data/Table.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div :class="wrapperClass" v-bind="attrs">
+  <div :class="[ui.wrapper, 'overflow-x-auto']" v-bind="attrs">
```

Comment:
You can't remove the `wrapperClass` here. Why not add the `overflow-x-auto` in the `wrapper` key of the `app.config`? https://github.com/nuxtlabs/ui/blob/dev/src/runtime/app.config.ts#L4

Reply:
I wasn't trying to remove the wrapperClass, I was only trying to add to it. I couldn't find where 'ui' is being called from, so I decided to add to the class. 

 But your correction makes more sense.

I have pushed a PR to make this correction [https://github.com/nuxtlabs/ui/pull/609](url)